.gitignore

```diff
@@ -17,6 +17,7 @@
 .idea/*
 .vscode
 .DS_Store
+.openzeppelin
```

Comment:
Don't ignore this, you can ignore it for the local hardhat chain, but make sure to commit for testnet and mainnet. 
Openzeppelin is using this folder to make sure that you do not introduce a storage collision bug.